src/schemathesis/cli/callbacks.py

```diff
@@ -1,14 +1,18 @@
 import pathlib
+import re
+import sys
 from contextlib import contextmanager
-from typing import Dict, Generator, Optional, Tuple
+from typing import Any, Dict, Generator, Optional, Tuple
 from urllib.parse import urlparse
 
 import click
 import hypothesis
 
+from schemathesis import utils
+
 
 def validate_schema(ctx: click.core.Context, param: click.core.Parameter, raw_value: str) -> str:
-    if not urlparse(raw_value).netloc:
+    if "app" not in ctx.params and not urlparse(raw_value).netloc:
```

Comment:
How the schema path should be validated if app is present? Could it be a usual validation? Or somehow adjusted - e.g we can allow full urls to load them via network and paths to load from the app itself